dali/test/python/operator_2/test_squeeze.py

```diff
@@ -71,7 +71,7 @@ def test_squeeze():
         ),
         (None, "Z", "XYZ", [(300, 1, 1), (10, 1, 1)], [(300, 1), (10, 1)], "XY"),
         (None, "ZY", "XYZ", [(300, 1, 1), (10, 1, 1)], [(300,), (10,)], "X"),
-        ([0], None, "X", [(1)], [()], ""),
+        ([0], None, "X", [1], [()], ""),
```

Comment:
Didn't we mean:
```suggestion
        ([0], None, "X", [(1,)], [()], ""),
```

Reply:
TBH, I don't know. Definably [(1)] != [(1,)].